fastlane/lib/fastlane/actions/set_info_plist_value.rb

```diff
@@ -10,7 +10,11 @@ def self.run(params)
         begin
           path = File.expand_path(params[:path])
           plist = Plist.parse_xml(path)
-          plist[params[:key]] = params[:value]
+          if params[:subkey]
+            plist[params[:key]][params[:subkey]] = params[:value]
```

Comment:
`key` is mandatory. `subkey` is optional. if no `key` is provided, it asks automagically on console prompt, at least in my test.